src/cc/CMakeLists.txt

```diff
@@ -32,8 +32,13 @@ if (CMAKE_COMPILER_IS_GNUCC)
   endif()
 endif()
 
-add_library(bcc SHARED bpf_common.cc bpf_module.cc libbpf.c perf_reader.c shared_table.cc exported_files.cc)
-set_target_properties(bcc PROPERTIES VERSION ${REVISION_LAST} SOVERSION 0)
+add_library(bcc-shared SHARED bpf_common.cc bpf_module.cc libbpf.c perf_reader.c shared_table.cc exported_files.cc)
+set_target_properties(bcc-shared PROPERTIES VERSION ${REVISION_LAST} SOVERSION 0)
+set_target_properties(bcc-shared PROPERTIES OUTPUT_NAME bcc)
+
+add_library(bcc-loader-static libbpf.c perf_reader.c)
```

Comment:
There's a dependency (bpf_map_create) required by both bcc and clang_frontend. This gets resolved at final link-time.